interface/code_systems/dataloads_ajax.php

```diff
@@ -231,16 +229,15 @@
                 success: function(data) {
                     $(stg_load_id).hide();
                     $(stg_dets_id).html(data);
-                    $(`#${dbName}_instrmsg`).hover( function() {
-                        dlgopen(`${dbName.toLowerCase()}_howto.php`, '', 800, 250, false, `<div class='text-warning'>${dbName} <?php echo xla("Installation Details"); ?></div>`, {
+                    $(document).ready(function() {});
+                    $(`#${dbName}_instrmsg`).click( function() {
+                        dlgopen(`${dbName.toLowerCase()}_howto.php`, '', 800, 250, false, `${dbName} <?php echo xla("Installation Details"); ?>`, {
```

Comment:
I agree with use click to replace hover,
hover on the tooltip then shows modal is very strange.

Reply:
yeah it's a problem when the hover is not intended.